Add coauthors changes (#73)
greenelab/phenoplier_manuscript: content/04.05.00.results_framework.md

![
**Schematic of the PhenoPLIER framework.**
**a)** High-level schematic of PhenoPLIER (a gene module-based method) in the context of TWAS (single-gene) and GWAS (single-variant).
In GWAS, we identify variants associated with traits.
In TWAS, first, we identify variants that are associated with gene expression levels (eQTLs); then, prediction models based on eQTLs are used to impute gene expression, which is used to compute gene-trait associations.
Resources such as LINCS L1000 provide information about how a drug perturbs gene expression; at the bottom-right corner, we show how a drug downregulates two genes (A and C).

> In PhenoPLIER, all these data types are integrated using groups of genes co-expressed in specific conditions (such as cell types) that we call gene modules or latent variables/LVs.

Reviewer: ```suggestion
In PhenoPLIER, these data types are integrated using groups of genes co-expressed across one or more unlabeled conditions (such as cell types) that we call gene modules or latent variables/LVs.
```
Trying to get the idea in here that the conditions exist but are unknown - I almost said latent conditions, but I think unlabeled -> latent variables might get that connection as well...
